### DISCUSSION TITLE
how to mention with chinese by use useMention hooks

### DISCUSSION THREAD

**Discussion by Just-focus** [remirror/remirror#1218] [Q&A]
![image](https://user-images.githubusercontent.com/18374969/132307098-00845e23-ecb5-4595-b4b7-26baacada185.png)

it can not trigger the state to still the mention action

**Comment by ocavue** [comment]
@Just-focus Hi. Could you try to add `supportedCharacters: /\S+/` to `MentionExtension`, for example:

```ts
      new MentionExtension({
        extraAttributes: { type: 'user' },
        matchers: [
          { name: 'at', char: '@', appendText: ' ', matchOffset: 0, supportedCharacters: /\S+/ },
        ],
      })
```

This should fix the issue. I will post a detailed example later. 


https://user-images.githubusercontent.com/24715727/132309730-090c1006-1f50-4a3e-82c9-f8eefdc93c93.mp4

**Reply by Just-focus** [reply]
thanks!

**Comment by ocavue** [comment]
@Just-focus Added a live demo here: https://pr1219-remirror-ocavue.vercel.app/?path=/story/react-hooks-usemention--more-supported-characters

You can check the source code at the bottom of this page.
